pkg/discovery/summary_test.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright the Sonobuoy contributors 2022
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package discovery
+
+import (
+	"testing"
+        "encoding/json"
+	"bytes"
+	"flag"
+        "path/filepath"
+	"io/ioutil"
+)
+
+const (
+	//Root directory where all the file sin the tarball are located
+	//Simulates the directory where all the files resulting from discovery end up before being compressed into the tarball
+	//For the test, this directory is the location where the test files are located
+	tarballRootDir = "testdata"
```

Comment:
This is why I wanted you to nest the files down an extra layer. Otherwise the next test that gets added has to 
 - realize they need to nest each case
 - go back and nest these files into a folder so it doesnt pollute their test

I'd rather the next time we add tests like this we can just copy the pattern we used here and specify the tarball root in each test case (since it will be test specific)